core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -270,7 +271,7 @@ private[spark] class BlockManager(
     // Don't reject broadcast blocks since they may be stored during task exec and
     // don't need to be migrated.
     if (isDecommissioning() && !blockId.isBroadcast) {
-        throw new BlockSavedOnDecommissionedBlockManagerException(blockId)
```

Comment:
I think the principle is we should not change the exception class?

Reply:
this exception class is only used once, I don't know should we keep it or not?

Reply:
Good catch. Yes let's keep the original exception classes and avoid changing them during the grouping process. We can make separate PRs in the future to improve quality.

Reply:
Done!